.github/workflows/pypi-release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     # We do this, since failures on test.pypi aren't that bad
     - name: Publish to Test PyPI
       if: startsWith(github.event.ref, 'refs/tags') || github.event_name == 'release'
-      uses: pypa/gh-action-pypi-publish@master
+      uses: pypa/gh-action-pypi-publish@v1.4
```

Comment:
this is failing on master

Reply:
@Borda mind having a look?

Reply:
it was working herehttps://github.com/PyTorchLightning/pytorch-lightning/actions/runs/345388960

Reply:
fixing here https://github.com/PyTorchLightning/pytorch-lightning/pull/4516